Source/GSThemeDrawing.m

```diff
@@ -2624,14 +2625,17 @@ - (void) drawScrollViewRect: (NSRect)rect
 	case NSLineBorder:
 	  [color set];
 	  NSFrameRect(bounds);
+          borderWidth = 1;
```

Comment:
We need to use [self sizeForBorderType: borderType] here and later use the correct component (width or height) as the theme may override the actual border drawing.

Reply:
I've updated the patch.